packages/mobile/src/app/reducers.ts

```diff
@@ -46,6 +47,9 @@ export interface State {
   multiTokenUseUpdatedFeed: boolean
   linkBankAccountEnabled: boolean
   sentryTracesSampleRate: number
+  supportedBiometryType: BIOMETRY_TYPE | null
+  biometryEnabled: boolean
+  useBiometry: boolean
```

Comment:
i think i will remove this and instead use the PincodeType to deduce if biometry is used in the next PR